Constitution.txt

```diff
@@ -1,490 +1,264 @@
+4: It is the Right of all People to petition their Government for redress of grievances; and the Responsibility of all People to watch their Government and ensure its good behavior, and to attempt to peaceably resolve intrusions by Government on the Rights of the People, before rising in arms against such an intrusion; and to rise in arms against the Government when peaceful resolutions become impossible.
 
-4: The Congress may by law provide for the case of the death of any of the persons from whom the House of Representatives may choose a President whenever the right of choice shall have devolved upon them, and for the case of the death of any of the persons from whom the Senate may choose a Vice President whenever the right of choice shall have devolved upon them.   affects 9
+5: It is the Right of all People to practice their religion freely and openly; and the Responsibility of all People to avoid interfering in the religious practice of others, and, in the practice of their religion, to respect the Rights of other People.
 
-5: Sections 1 and 2 shall take effect on the 15th day of October following the ratification of this article.
+6: It is the Right of all People to be considered innocent of any crime, unless such crime shall be proven in a court of law, as decided by unanimous decision of a jury of at least twelve disinterested peers, and for such a trial to be carried out with reasonable speed in public, and for the result of such a trial to be respected in perpetuity; and to have access to legal Representation provided by the court or the prosecution, when the accused Person is unable to retain such Representation by their own means; and to be reasonably compensated for time served on a jury; and the Responsibility of all People to serve on such juries when called upon, and in such service adhere to the Truth, the Rights of the People, and the Law; and in such cases as the Law is in conflict with Truth or the Rights of the People, to judge in favor of Truth and the Rights of the People, against that unjust Law.
 
-6: This article shall be inoperative unless it shall have been ratified as an amendment to the Constitution by the legislatures of three-fourths of the several States within seven years from the date of its submission.   ratified #20
+Article VI (Article 6 - Limitations on the Government with respect to the Rights of the People)
```

Comment:
This Article is an incorporation of the majority of the Bill of Rights. Again I note that it stands separate from the Declaration of Rights and Responsibilities. We are back into the delineation of government authority, having finished with the philosophy.